etc/pip-requires-test

```diff
@@ -24,4 +24,4 @@ subprocess32==3.5.4; python_version <= '3.0'      # A backport of the subprocess
 pycodestyle==2.5.0                                # New package replacing pep8
 mock==3.0.5; python_version < '3.3'               # Mock package needed for readthedocs build for mocking pycurls (mock seems to break SQLAlchemy in Python 3 tests + since Python 3.3 there is unittest.mock)
 s3cmd==2.0.2                                      # Used for testing of S3
-PyYAML==5.1.1                                     # Used for reading test configuration files (also in globus_extras)
+PyYAML==5.4                                       # Used for reading test configuration files (also in globus_extras)
```

Comment:
The comment says it… have you looked into the `setup.py` ? also `readthedocs.requirements.txt`

Reply:
👍 Thanks, this indeed needs to go to the extras in setup.py too. Will add.
For the readthedocs file, that update is usually part of the build procedure, so it would be updated then.